Civi/CCase/Utils.php

```diff
@@ -21,18 +21,20 @@ public static function getRelationshipTypesListByCaseRole($roleName) {
     ));
     $relationshipTypes = civicrm_api3('RelationshipType', 'get', array(
       'options' => array('limit' => 0),
-      'return' => array('name_b_a'),
+      'return' => array('name_b_a', 'name_a_b'),
     ));
-    $relationshipTypes = \CRM_Utils_Array::rekey($relationshipTypes['values'], 'name_b_a');
+    $relationshipTypes = $relationshipTypes['values'];
 
     foreach ($caseTypes['values'] as $caseType) {
       $caseTypeToCaseRolesList = array();
+
       foreach ($caseType['definition']['caseRoles'] as $role) {
-        if ($roleName == 'involved') {
-          $caseTypeToCaseRolesList[] =  $relationshipTypes[$role['name']]['id'];
-        } else {
-          if (!empty($role[$roleName])) {
-            $caseTypeToCaseRolesList[] = $relationshipTypes[$role['name']]['id'];
+        if ($roleName == 'involved' || !empty($role[$roleName])) {
+          foreach ($relationshipTypes as $key => $value) {
```

Comment:
```php
foreach ($relationshipTypes as $relationshipType) {
```

- We are not using the `$key` var so there's no need to define it.
- `$value` is too generic and should be avoided if possible.